.buildkite/pipeline.yml

```diff
@@ -144,3 +144,4 @@ steps:
           CLIMACORE_DISTRIBUTED: "MPI"
         agents:
           slurm_ntasks: 32
+          slurm_mem: 80GB
```

Comment:
Should this be done separately?

Reply:
Yeah, probably. I just didn't want to merge with main before all the CI tests were passing, but I guess `MPI AMIP FINE` is already failing in main too.

Reply:
Maybe just do this in a separate PR to fix CI (see the [latest build](https://buildkite.com/clima/climacoupler-ci/builds/586#0184fa24-d737-4ab9-8a92-4e053d5d4f70)). It's a bit striking that we need this much memory, cc @simonbyrne

Reply:
Yeah, I think we should first focus on fixing CI, then these other changes should be able to merge in smoothly

Reply:
I'm not sure what the issue is, but do you open the netcdf file on every MPI process?